src/components/platformGrid.tsx

```diff
@@ -79,7 +79,12 @@ export default ({ noGuides = false }: Props): JSX.Element => {
               <PlatformCell>
                 <PlatformCellIcon>
                   <SmartLink to={platform.url}>
-                    <PlatformIcon size={82} platform={platform.key} />
+                    <PlatformIcon
+                      size={82}
+                      platform={platform.key}
+                      format="lg"
+                      style={{ maxWidth: "none", border: 0, boxShadow: "none" }}
```

Comment:
is this due to default styles in PlatformIcon, or due to stuff its inheriting in docs?

Reply:
Due to stuff its inheriting. The old local platformIcon component used to apply it everywhere:

https://github.com/getsentry/sentry-https://github.com/getsentry/sentry-docs/blob/860cd098a2b7536946f40ac786b2f0defd10c36a/src/components/platformIcon.tsx#L82-L90

but this is the only place where it makes a real difference.